docs/source/workflows/observe/observe-workflow-with-dynatrace.md

```diff
@@ -0,0 +1,118 @@
+    * `openTelemetryTrace.ingest`
+    * `metrics.ingest`
+    * `Logs.ingest`
+1. Select Generate token.
+1. Copy the generated token to the clipboard. Store the token in a password manager for future use and for the configuration below.
+
+## Step 3: Configure OTel Collector
+
+Configure an OTel Collector configuration file using an `otlphttp` exporter to the Dynatrace OTLP API as shown in the example below. Refer to the [Dynatrace documentation](https://docs.dynatrace.com/docs/shortlink/otel-collector-config) as required.
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+processors:
+  cumulativetodelta:
+exporters:
+  otlphttp:
+    endpoint: "https://<YOUR-DYNATRACE-ENVIRONMENT>.live.dynatrace.com/api/v2/otlp"
+    headers:
+      Authorization: "Api-Token <YOUR-DYNATRACE-TOKEN>"
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: []
+      exporters: [otlphttp]
+    metrics:
+      receivers: [otlp]
+      processors: [cumulativetodelta]
+      exporters: [otlphttp]
+    logs:
+      receivers: [otlp]
+      processors: []
+      exporters: [otlphttp]
+```
+
+## Step 4: Install and run your configured OTel Collector
+
+There are many ways to deploy an [OTel Collector](https://docs.dynatrace.com/docs/ingest-from/opentelemetry/collector/deployment) but for this example, an OTel Collector is created using Docker with the configuration from the previous step into a file named `otelcollectorconfig.yaml` with the [Dynatrace distribution of the OpenTelemetry Collector](https://docs.dynatrace.com/docs/ingest-from/opentelemetry/collector).  
+
+```bash
+docker run -d -v $(pwd)/otelcollectorconfig.yaml:/etc/otelcol/config.yaml \ 
+-p 4318:4318 \ 
+dynatrace/dynatrace-otel-collector:0.34.0 
+```
+
+Once running, the collector endpoint is: `http://localhost:4318`. 
+
+## Step 5: Install the NVIDIA NeMo Agent toolkit OpenTelemetry Subpackages
+
+```bash
+# Install all optional telemetry extras
+uv pip install -e '.[telemetry]'
+
+# Install specific telemetry extras
+uv pip install -e '.[opentelemetry]'
+```
+
+## Step 6: Modify NVIDIA NeMo Agent toolkit Workflow Configuration
+
+Update your workflow configuration file to include the telemetry settings.
+
+Example configuration:
+```yaml
+general:
+  telemetry:
+    tracing:
+      otelcollector:
+        _type: otelcollector
+        # The endpoint where you have deployed the otel collector
+        endpoint: http://0.0.0.0:4318/v1/traces
+        project: your_project_name
+```
+
+## Step 7: Run the workflow
+
+As workflows run, spans are sent to the OTel Collector which in turn exports them based on the exporter you configured to Dynatrace. 
+
+In the example below, the [Simple LangSmith-Documentation Agent workflow](https://github.com/NVIDIA/NeMo-Agent-Toolkit/tree/release/1.2/examples/getting_started/simple_web_query) was used.
```

Comment:
We cannot hardcode versioned examples because we will forget to bump for the next release. It's also a chicken-egg problem. I would recommend removing this sentence and use the same language as the other observability examples.

Reply:
will address, not a problem